.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: Close stale PRs
+
+on:
+  schedule:
+  - cron: "0 * * * *"
```

Comment:
This would run every minute? `0 0 * * 0` for weekly maybe?

Reply:
`0 * * * *` is [on the hour every hour](https://crontab.guru/#0_*_*_*_*). How about we set to to run daily, at least at the beginning, so that it can process the backlog in a decent amount of time? There is a [rate limit](https://github.com/actions/stale/blob/0d6a8e531449ecb97c9b6046b6b5d8eeac8b5115/action.yml#L28-L30) on actions taken.